reviews/utils/review_utils.py

```diff
@@ -304,6 +304,33 @@ def notify_secretary(decision):
     msg_plain = render_to_string('mail/decision_notify.txt', params)
     send_mail(subject, msg_plain, settings.EMAIL_FROM, [secretary.email])
 
+def notify_secretary_all_decisions(review):
+    """
+    Notifies a secretary all Decisions have been made for a certain review
+    """
+    # Change language to Dutch for this e-mail, but save the current language to reset it later
+    current_language = get_language()
+    activate("nl")
+
+    secretary = get_secretary()
+    subject = "FETC-GW {}: alle beoordelingen toegevoegd".format(
+        review.proposal.committee_prefixed_refnum(),
+    )
+    params = {
+        "secretary": secretary.get_full_name(),
+        "review": review,
+        "decisions": review.decision_set.all(),
+        "review_detail_page": settings.BASE_URL
+        + (reverse("reviews:detail", args=[review.pk])),
+        "close_review_page": settings.BASE_URL
+        + (reverse("reviews:close", args=[review.pk])),
+    }
+    msg_plain = render_to_string("mail/all_decisions_notify.txt", params)
+    send_mail(subject, msg_plain, settings.EMAIL_FROM, ["fetc-gw@uu.nl"])
+
```

Comment:
I should've clarified; you should use `settings.EMAIL_FROM` as the recipient.

It's kinda weird, I know, but `EMAIL_FROM` is the NPM email. Hardcoding this is a bad idea, as names (and thus, emails) in the UU tend to change. It's better if we only have to change the one value when that happens